website/docs/reference/change-requests.md

```diff
@@ -62,7 +63,43 @@ From here, you can navigate to the change request overview page. This page will
 
 ![Change request banner](/img/change-request-overview.png)
 
-From here, if you have the correct permissions, you can approve and apply the change request. Once applied the changes will be live in production.
+From here, if you have the correct permissions, you can approve and schedule or apply the change request. Once applied, the changes will be live in production.
+
+### Scheduled changes
+
+:::info Availability
+
+Change request scheduling is currently in development and will be released in an upcoming version of Unleash. How the feature works (and as such, the contents of this subsection) can change before the feature is released.
+
+:::
+
+When a change request is approved, you can schedule it to be applied at a later time. This allows you to group changes together and apply them at a time that is convenient for you, such as during a maintenance window, or at a time when you know there will be less traffic to your application.
+
+Scheduled changes can be rescheduled, applied immediately, or rejected. They can not be edited or moved back to any of the previous states.
+
+Unleash will attempt to apply the changes at the scheduled time. However, if there are conflicts, the changes will not be applied and the change request will be marked as failed. Conflicts can occur if the change request contains changes that conflict with changes that have already been applied to the environment. Such as if the change request tries to update a flag or a strategy that has been deleted.
+
+Unleash will warn you ahead of time if you make changes that conflict with a scheduled change request.
+
+If Unleash fails to apply a scheduled change request, the change request will remain in the scheduled state. You can reschedule it and try to apply it again later, or you can reject it.
+
+If a scheduled change request can not be applied, Unleash will send a notification to the person who scheduled it and to the person who created the change request. Unleash will also send an email to the person who scheduled it.
+
+When a scheduled change request is applied, the person who scheduled it will receive a notification.
+
+#### Application rights
+
+If the user who scheduled a change request is deleted from the Unleash users list before the scheduled time, the changes will **not** be applied.
```

Comment:
Is this implemented?

Reply:
I don't know, but I think this is part of what we agreed on. But maybe we still need to do some user research there? But also, that's a god point about change requests being disabled for the environment: I'll make a not of that.

Reply:
@andreas-unleash Where did we land on this one?

Reply:
After a discussion: we'll implement this before going into beta. It's a business decision that can be reversed later, but for now it's a safer place to start.